crates/net/eth-wire/src/types/broadcast.rs

```diff
@@ -68,7 +68,6 @@ impl From<NewBlockHashes> for Vec<BlockHashNumber> {
 
 /// A new block with the current total difficulty, which includes the difficulty of the returned
 /// block.
-#[derive_arbitrary(rlp, 25)]
```

Comment:
why was this replaced with manual arbitrary impls? shouldn't this derive if `Arbitrary` is implemented?

Reply:
Yeah, I am trying to understand why this is the case. I manually checked , it indeed calls arbitrary_with impl of `Block`. 
Tests pass if I use a manual impl for `NewBlock` also, but fail otherwise.
Looking into it